src/win/win_path_manager.h

```diff
@@ -5,12 +5,16 @@
 #include "i_path_manager.h"
 
 class WinPathManager : public IPathManager {
+    private: 
+        const char *appdata = getenv("APPDATA");
+        std::string path = appdata;        

```

Comment:
I would suggest to rename `path` to `appdata` and directly asigning the result of `getenv`.
Furthermore it would be good to make appdata publicly accessible.

Reply:
directly assigning is not possible. #getenv() returns a char array, so it has to be initialized as a char. Afterwards it can be converted to a string.

Reply:
`std::string appdata = getenv("APPDATA");` should work fine